packages/pwa-buildpack/lib/Utilities/graphQL.js

```diff
@@ -10,7 +14,8 @@ const fetchQuery = query => {
                 'Content-Type': 'application/json',
                 'Accept-Encoding': 'gzip'
             },
-            body: JSON.stringify({ query })
+            body: JSON.stringify({ query }),
+            agent: fetchAgent
```

Comment:
This [SO post](https://stackoverflow.com/questions/31861109/tls-what-exactly-does-rejectunauthorized-mean-for-me) suggests that the better solution is to use a custom CA. This may not be as easy for people who might want to use PWA Studio directly out of the box without much customization. What do you think?

Reply:
There's a comment thread in the original issue to research/implement that solution as a followup, I fully agree. Woes of splitting off to JIRA.

https://github.com/magento/pwa-studio/issues/1885#issuecomment-552658990

Reply:
Seems like we should make a ticket for that in JIRA. Regardless I'll approve it :)

Reply:
😎 
https://jira.corp.magento.com/browse/PWA-158